data-structures-linked-lists/append.js

```diff
@@ -0,0 +1,11 @@
+/* eslint-disable no-unused-vars */
+import LinkedList from './lib/linked-list.js';
+
+export default function append(list, value) {
+  const newNode = new LinkedList(value);
+  while (list.next !== null) {
+    list = list.next;
+  }
+  list.next = newNode;
+  return list;
```

Comment:
This actually returns the last node in the original list before you appended. But, the spec says you shouldn't return anything, so I guess this is OK!